.github/workflows/check-catalyst.yaml

```diff
@@ -520,16 +528,6 @@ jobs:
         key: Linux-qir-stdlib-build
         fail-on-cache-miss: True
 
-    - name: Build Catalyst-Runtime
-      run: |
-        COMPILER_LAUNCHER="" \
-        ENABLE_OPENQASM=ON \
-        RT_BUILD_DIR="$(pwd)/runtime-build" \
-        QIR_STDLIB_DIR="$(pwd)/qir-stdlib-build" \
-        QIR_STDLIB_INCLUDES_DIR="$(pwd)/qir-stdlib-build/include" \
-        CMAKE_ARGS="-DENABLE_OPENMP=ON" \
```

Comment:
This line stood out to me as well, do we use OpenMP in the OpenQASM backend? 😅

Reply:
Hehe, that's on our roadmap :smile: 

This was originally added to test `CMAKE_ARGS` and building/testing Lightning tests using OMP. This can be deprecated though as Lightning's default build is OMP=ON and the cmake flag is tested in other places.